scripts/ci/complianceoperator/create.sh

```diff
@@ -2,13 +2,7 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
 
-echo "Deploying Compliance Operator"
-git clone git@github.com:openshift/compliance-operator.git
-cd compliance-operator
-
-oc create -f deploy/ns.yaml
-for f in $(ls -1 deploy/crds/*crd.yaml); do oc apply -f $f -n openshift-compliance; done
-oc apply -n openshift-compliance -f deploy/
+echo "Validating the Compliance Operator is ready for use"
```

Comment:
Note: we could improve the check below to check for the existence of the ProfileBundle and that it's in a `Valid` state. Once it is, the profile and rules should both be available.